docs/docs/reference/migration.md

```diff
@@ -36,7 +36,13 @@ done
 echo "All specified rules have been deleted."
 ```
 
-## Migrations to v2.19.0
+## Migrating from CLI versions before 2.21.1
+
+### AWS
+
+* AWS clusters that utilize `LoadBalancer` resources require additional IAM permissions. Please upgrade your IAM roles using `iam upgrade apply`. This will show necessary changes and apply them, if desired.
```

Comment:
```suggestion
* AWS clusters that utilize `LoadBalancer` resources require additional IAM permissions. Please upgrade your IAM roles using `constellation iam upgrade apply`. This will show necessary changes and apply them, if desired.
```
?